packages/apollo-server-types/src/index.ts

```diff
@@ -84,6 +84,11 @@ export interface ApolloConfig {
   };
 }
 
+export interface GraphQLSchemaContext {
+  apiSchema: GraphQLSchema,
+  coreSchemaSdl?: string,
```

Comment:
Can you leave a comment noting that this will only be left out with old versions of Gateway and that we should consider making this required for AS3 (which can definitely have a minimum version of Gateway)? (Though my long comment suggests we can probably make this required now.)

Reply:
This argument may still be gone I think, specifically if `schemaDidLoadOrUpdate` is used by a plugin when a gateway isn't provided (which is the case for non-federated schema reporting).